src/client/components/UserPage/Drawer/ControlPanel/DrawerHeader.tsx

```diff
@@ -26,10 +33,12 @@ const useStyles = makeStyles(() =>
 export default function DrawerHeader() {
   const classes = useStyles()
```

Comment:
For mobile view, I believe there needs to be more padding between the close icon and the copy link button.